lib/classes/calculator_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+class Calculator
+  
+  def add(list_of_numbers)
+    list_of_numbers.sum
+  end
+  
+  def max(list_of_numbers)
+    list_of_numbers.max
+  end
+  
+  def min(list_of_numbers)
+    list_of_numbers.min
+  end
+  
+  def odd(list_of_numbers)
+    list_of_numbers.sum.odd?
+  end
+  
+  def even(list_of_numbers)
+    list_of_numbers.sum.even?
+  end
+  
+  def self.hello
+   "Hello, I am your Calculator :)"
+  end
+end
+
+describe Calculator do 
+  let(:calculator) {Calculator.new}
+  
+  
```

Comment:
minor thing 

use 1 empty line, there is no need for 2 empty lines, 
on real projects this will be commented :P and tools for styling such as rubocop will complain for this